spec/models/ability_spec.rb

```diff
@@ -143,29 +145,28 @@
         end
       end
 
+      describe "team's students or admin should be able to mark attendance to a conference" do
+        context 'when user is a student from a team and try to update attendance' do
 
-      describe "a user should not be able to mark another's attendance to a conference" do
-
-        context 'when same user' do
-          let!(:attendance) { FactoryGirl.create(:attendance, user: user)}
+          let!(:user) { FactoryGirl.create(:student) }
+          let!(:attendance) { FactoryGirl.create(:attendance, team_id: user.student_team.id) }
 
           it 'allows marking of attendance' do
-            expect(ability).to be_able_to(:crud, attendance)
+            expect(ability).to be_able_to(:crud, user)
```

Comment:
No worries 😉 
Just tell me if you need any more information on how these tests are set up.